pkg/detectors/railwayapp/railwayapp.go

```diff
@@ -0,0 +1,172 @@
+package railwayapp
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	regexp "github.com/wasilibs/go-re2"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct {
+	client *http.Client
+}
+
+// graphQLResponse will handle the response from railway API
+type graphQLResponse struct {
+	Data   interface{}   `json:"data"`
+	Errors []interface{} `json:"errors"`
+}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	defaultClient = common.SaneHttpClient()
+
+	apiToken = regexp.MustCompile(detectors.PrefixRegex([]string{"railwayapp", "railway"}) +
```

Comment:
```suggestion
	apiToken = regexp.MustCompile(detectors.PrefixRegex([]string{"railway"}) +
```

**Suggestion**: Since `railway` is a substring of `railwayapp`, we can remove the check for `railwayapp`. Any occurrence of `railwayapp` would get caught by just `railway` Same goes for the regex.